VideoLocker/res/layout/row_course_dashboard_cert.xml

```diff
@@ -14,6 +14,7 @@
         <ImageView
             android:layout_width="wrap_content"
             android:layout_height="match_parent"
+            android:contentDescription="@null"
```

Comment:
This image is shown in the courseware page when the user earns a certificate. The image of the certificate is not Talkbalk focusable so the content description can be null.